modules/admin_manual/pages/installation/docker/index.adoc

```diff
@@ -47,6 +47,10 @@ configuration settings. Only a few settings are required, these are:
 | `{std-port-http}`
 |===
 
+Note: `ADMIN_USERNAME` and `ADMIN_PASSWORD` will not change between deploys even if you change the 
```

Comment:
```suggestion
NOTE: `ADMIN_USERNAME` and `ADMIN_PASSWORD` will not change between deploys even if you change the 
```
Needs correct writing because `NOTE:` is a antora macro creating a proper note.